modin/data_management/factories/dispatcher.py

```diff
@@ -35,32 +62,48 @@ def stub(*args, **kw):
 
 class StubFactory(factories.BaseFactory):
     """
-    A factory that does nothing more than raise NotImplementedError when any method is called.
+    Factory that does nothing more than raise NotImplementedError when any method is called.
+
+    Notes
+    -----
     Used for testing purposes.
     """
 
     io_cls = StubIoEngine()
 
     @classmethod
     def set_failing_name(cls, factory_name):
+        """
+        Fill in `.io_cls` class attribute with ``StubIoEngine`` engine.
+
+        Parameters
+        ----------
+        factory_name : str
+            Name to pass to the ``StubIoEngine`` constructor.
+        """
         cls.io_cls = StubIoEngine(factory_name)
         return cls
 
 
 class EngineDispatcher(object):
     """
-    This is the 'ingestion' point which knows where to route the work
+    Class that routes IO-work to the engines.
+
+    This class is responsible for keeping selected engine up-to-date and dispatching
+    calls of IO-functions to its actual engine-specific implementations.
     """
 
     __engine: factories.BaseFactory = None
 
     @classmethod
     def get_engine(cls) -> factories.BaseFactory:
+        """Get current execution engine."""
         # mostly for testing
         return cls.__engine
 
     @classmethod
-    def _update_engine(cls, _):
+    def _update_engine(cls, _):  # noqa: PR01
```

Comment:
Why is `#noqa: PR01` used?

Reply:
`PR01: missing docstring for _ parameter` do we need to document `_` parameter?

Reply:
No, we needn't to. But what do we use it for?

Reply:
It seems that it's used in the glory of compatibility. We're subscribing this function on updates of `Engine` and `Backend` env-vars. When this function is invoked, subscription-manager passes itself as a function argument [[link]](https://github.com/modin-project/modin/blob/5aa59582fc3d2fc73e7f04f6d486e118915ac89f/modin/config/pubsub.py#L174), which becomes `_` in `_update_engine` since we don't use it. I'll leave a note about it in docs, and make changing `_ -> *args` one of a point in #3008

Reply:
Okay, I see. Probably, it makes sense to leave as it is.